yellowbrick/features/jointplot.py

```diff
@@ -447,5 +447,120 @@ def _index_into(self, idx, data):
 ##########################################################################
 
 
-def joint_plot():
-    raise NotImplementedError("quick method still needs to be implemented")
+def joint_plot(
+        X,
+        y,
+        ax=None,
+        columns=None,
+        correlation="pearson",
+        kind="scatter",
+        hist=True,
+        alpha=0.65,
+        joint_kws=None,
+        hist_kws=None,
+        show=True,
+        **kwargs
+):
+    """
+        Joint plots are useful for machine learning on multi-dimensional data, allowing for
```

Comment:
It looks like there is one extra indent for the docstring, would you mind unindenting it and the final `"""` one level?

Reply:
the indent issue has been fixed.